package.json

```diff
@@ -6,7 +6,7 @@
     "start": "node ./bin/www",
     "dev": "nodemon ./bin/www",
     "testDB": "sequelize db:drop --env test && sequelize db:create --env test && sequelize db:migrate --env test && sequelize db:seed:all --env test",
-    "test": "npm run testDB && mocha",
+    "test": "npm run testDB",
```

Comment:
If the "test" script doesn't call `mocha`, it won't test the endpoints. It will just prepare the database to be tested. I suggest removing "test" script, renaming "mocha" script to "test" and also renaming "testDB" script to "pretest" ([Pre Scripts](https://docs.npmjs.com/cli/v8/using-npm/scripts#pre--post-scripts))